ui/src/components/executions/Overview.vue

```diff
@@ -163,7 +164,19 @@
                 return ret;
             },
             inputs() {
-                return toRaw(this.execution.inputs);
+              if (!this.flow) {
+                return []
+              }
+
+              let inputs = toRaw(this.execution.inputs);
+              Object.keys(inputs).forEach(key => {
+                this.flow.inputs.forEach(input => {
+                  if(key === input.name && input.type === 'SECRET') {
+                    inputs[key] = '******';
```

Comment:
This should be done on the backend as the secret would still be store in local

Reply:
If we do it in the backend the input will not be useable.